### PR TITLE
Added PWA capability and updated styling

### PR DESCRIPTION
**Pull request by btroncone**
Some updates to the project include:

- Added the necessary requirements for PWA including `@angular/service-worker`
- Added support for WallabyJS testing
- Updated project to use `scss`
- Updated config to prefix components with `rx`
- Small route config tweak for default routes

### CODE REVIEW

File: src/app/team/team.component.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { Component, OnInit } from '@angular/core';
 
 @Component({
-  selector: 'app-team',
+  selector: 'rx-team',
```
Comment (ladyleet) [inline src/app/team/team.component.ts]: @btroncone i don't love changing the selectors or the conventions that the cli gives us - any good reason why?

File: src/index.html [hunk 2]
```diff
@@ -2,13 +2,19 @@
...
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="manifest" href="manifest.json">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Mono" rel="stylesheet">
 </head>
 <body>
-  <app-root></app-root>
+  <rx-root></rx-root>
```
Comment (ladyleet) [inline src/index.html]: same comments as last comment on changing default selectors

File: src/app/team/team.component.ts (hunk 1 above)
Comment (btroncone) [reply src/app/team/team.component.ts]: It's more habit than anything, I generally adjust prefix to be app specific. I'm perfectly fine with `app` though, I'll update real quick. 👍

**Comment by btroncone** [comment]
I updated the prefix back to `app` 👍

**Comment by ladyleet** [comment]
@btroncone woot!

**Pull request merged**